static/bidder-info/smilewanted.yaml

```diff
@@ -0,0 +1,12 @@
+maintainer:
+  email: "maxime@smilewanted.com"
```

Comment:
This looks like a personal email address. We require the use of a shared email group to be sure Prebid.org has a communication path if that individual leaves the company.